CADETProcess/comparison/difference.py

```diff
@@ -535,14 +539,21 @@ def __init__(
 
     @property
     def n_metrics(self):
+        n_metrics = 2
+
+        if self.include_height:
+            n_metrics += 1
+
         if self.use_derivative:
```

Comment:
Should we also only optionally compute peak height difference for the derivatives (i.e. if self.include_height)?

Reply:
I've changed it to do that.
```
        include_height : bool, optional
            If True, also return a metric for the height difference.
            If `use_derivative` is also true, also return height differences
            for the derivatives.
```